### ISSUE TITLE
Rereskillable Reforked Incompatibility

### ISSUE THREAD

**Issue by GhostLupine** [Yesssssman/epicfightmod#1085]
So I'm making a modpack and decided to include epic fight. But a problem arised after seeing that I can use a weapon with a level requirement I haven't met in epic fight mode. Is this issue fixable or can I not use this mod anymore, for the sake of making progression way too easy?

https://user-images.githubusercontent.com/98993704/179591017-e76cfd7a-48a8-4172-9093-eca9e4346bd8.mp4

**Comment by MetalKnight56** [comment]
Yeah this is to be expected since epic fight uses a different system for hit reg.

**Comment by MetalKnight56** [comment]
could you please provide epic fight,rereskillable reforked and minecraft version?

**Comment by Sazuzaki** [comment]
> could you please provide epic fight,rereskillable reforked and minecraft version?

I'll go ahead an reply for him as I tried testing this mod out too. 

Minecraft: 1.18.2
Epic Fight: 18.3.6
Rereskillable Rereforked: 2.1.0

**Comment by adoetho** [comment]
@Sazuzaki If you still want to use Epic Fight and an RPG mod, use RPGskillable. It allows you to apply a generic attack speed decrease if you lack requirements, which still functions with Epic Fight (as you're basically unable to attack or dodge until the animation completes, which takes around 10x longer than usual if you lack requirements).

**Comment by guivnf** [comment]
This issue should be fixed since we've changed the way epic fight uses it's hit interaction. Should now be compatible with vanilla.
